app/_src/gateway/support/index.md

```diff
@@ -45,6 +45,9 @@ Customers with platinum or higher subscriptions may request fixes outside of the
 Kong supports the following versions of {{site.ee_product_name}}: 
 
 {% navtabs %}
+  {% navtab 3.3 %}
```

Comment:
❗ Reviewers: The 3.3 nav tab renders correctly, but it's not showing the contents of the table. Anyone know what I did wrong?

Reply:
what do you mean? I'm seeing it fine here: https://deploy-preview-5536--kongdocs.netlify.app/gateway/latest/support-policy/

![Screenshot 2023-05-09 at 4 43 50 PM](https://github.com/Kong/docs.konghq.com/assets/54370747/ea875bf1-2978-4bc2-bf28-0060cd352cd7)

Reply:
Hmm, looks okay to me now in the preview, maybe it was a cache thing?